src/watchmaker/workers/salt.py

```diff
@@ -254,6 +255,12 @@ def _build_salt_formula(self, extract_dir):
                 filepath=salt_content_file,
                 to_directory=extract_dir
             )
+        else:
+            local_content = os.sep.join((
+                static.__path__[0], 'salt', 'content'))
+            if os.path.exists(local_content):
+                self.log.info('Using local salt content: %s', local_content)
+                distutils.dir_util.copy_tree(local_content, extract_dir)  # noqa: E501 # pylint: disable=no-member,line-too-long
```

Comment:
Why is pylint complaining about `no-member` here?

Reply:
I wish I knew! https://stackoverflow.com/questions/18908941/unable-to-import-distutils-dir-util-on-windows/33417555

Reply:
Another reason not to use `distutils` 😛